data.json

```diff
@@ -111,7 +111,7 @@
     "name": "Hacktoberfest 2019",
     "difficulty": "hard",
     "description": "To qualify for the official limited edition Hacktoberfest T-shirt, you must register and then make four pull requests (PRs) between October 1-31 (in any time zone).",
-    "reference": "https://hacktoberfest.digitalocean.com/" ,
```

Comment:
Theses changes are out of the scope of this PR, can you please revert them?

Reply:
Reverted the changes caused by my linter. Sorry!